site/en/setup.md

```diff
@@ -122,7 +131,36 @@ docker pull ocrd/all:maximum
 
 Replace `maximum` accordingly if you want the `minimum` or `medium` variant.
 
-(Also, if you want to keep the modules' git repos inside the Docker images – so you can keep making fast updates, without waiting for a new pre-built image but also without building an image yourself –, then add the suffix `-git` to the variant, e.g. `maximum-git`. This will behave like the native installation, only inside the container. Yes, you can also [commit changes](https://rollout.io/blog/using-docker-commit-to-create-and-change-an-image/) made in containers back to your local Docker image.)
+(Also, if you want to keep the modules' git repos inside the Docker images – so you can keep making 
+fast updates, without waiting for a new pre-built image but also without building an image yourself –, 
+then add the suffix `-git` to the variant, e.g. `maximum-git`. This will behave like the native installation, 
+only inside the container. Yes, you can also [commit changes](https://rollout.io/blog/using-docker-commit-to-create-and-change-an-image/) 
+made in containers back to your local Docker image.)
+
+### Testing the Docker installation
+
+For example, let's fetch a document from the [OCR-D GT Repo](https://ocr-d-repo.scc.kit.edu/api/v1/metastore/bagit/):
+
+```sh
+wget 'https://ocr-d-repo.scc.kit.edu/api/v1/dataresources/736a2f9a-92c6-4fe3-a457-edfa3eab1fe3/data/wundt_grundriss_1896.ocrd.zip'
+unzip wundt_grundriss_1896.ocrd.zip
+cd data
+```
+
+Let's segment the images in file group `OCR-D-IMG` from the zip file into regions (creating a
+first [PAGE-XML](https://github.com/PRImA-Research-Lab/PAGE-XML) file group
+`OCR-D-SEG-BLOCK-DOCKER`)
+
+You can spin up a docker container, mounting the current working directory like this:
+
+```sh
+docker run -u $(id -u) -w /data -v $PWD:/data -- ocrd/all:maximum ocrd-tesserocr-segment-region -I OCR-D-IMG -O OCR-D-SEG-BLOCK-DOCKER
```

Comment:
https://github.com/OCR-D/ocrd-website/issues/125 we should change to long options for docker calls for clarity